src/config.ts

```diff
@@ -91,13 +100,22 @@ export function getMergedConfig(): ConsentManagerConfig {
   config.privacyPolicy ??= privacyPolicy;
   config.secondaryPolicy ??= secondaryPolicy;
   config.dismissedViewState ??= dismissedViewState;
+  config.languages ??= languages;
+
+  // Determine the language settings to use
+  const existingLanguages = config.languages ? config.languages.split(',') : [];
```

Comment:
Do we want to allow optional whitespace?

```suggestion
  const existingLanguages = config.languages ? config.languages.split(/\s*,\s*/) : [];
```

Reply:
added - i normally do .split(',').map((x) => x.trim()).filter((x) => !!x) but this is probably more efficient lol